package-lock.json

```diff
@@ -79,6 +79,7 @@
         "nodemon": "^3.1.0",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
+        "react-router-dom": "^6.23.0",
```

Comment:
I'm a bit confused about how the dependencies were introduced for the file. I don't see any package.json changes in root folder